### ISSUE TITLE
`invalid face attribute: inherit nil` when using the navigator

### ISSUE THREAD

**Issue by mohkale** [emacs-dashboard/emacs-dashboard#191]
I've had this issue for a little while, but today I finally put in the effort to try and find out what's causing it. When ever I'm on the dashboard buffer, my messages buffer get's filled with:

```
Invalid face attribute :inherit nil
Invalid face reference: nil
``` 

repeating indefinitely. I've only found this to happen whilst the navigator is active, so I can only presume that's a prerequisite for the error.

To replicate, you can do the following:
- run `emacs -q`
- run `M-x package-initialize`
- run `M-: (require 'dashboard)`
- run the following in `M-:`
```emacs-lisp
(setq
 dashboard-set-navigator t
 dashboard-navigator-buttons
 `(;; line1
   ((,(all-the-icons-octicon "mark-github" :height 1.1 :v-adjust 0.0)
     "Homepage"
     "Browse homepage"
     (lambda (&rest _) (browse-url "homepage")))
    ("★" "Star" "Show stars" (lambda (&rest _) (show-stars)) warning)
    ("?" "" "?/h" #'show-help nil "<" ">"))
   ;; line 2
   ((,(all-the-icons-faicon "linkedin" :height 1.1 :v-adjust 0.0)
     "Linkedin"
     ""
     (lambda (&rest _) (browse-url "homepage")))
    ("⚑" nil "Show flags" (lambda (&rest _) (message "flag")) error))))
```
**Note:** this is just the navigator shown on the [README](https://github.com/emacs-dashboard/emacs-dashboard).
- run `C-x b *Messages*` and verify the messages buffer does not contain `invalid face attribute: inherit nil`.
- run `C-x b *dashboard*`
- run `M-x dashboard-refresh-buffer`
- run `C-x b *Messages*` and notice that the invalid face attribute messages have returned.

you can repeat this, without setting the navigator and you should have no problems. This seems to be a problem directly related to the navigator.

Doing some sleuthing, I believe this is caused by [this](https://github.com/emacs-dashboard/emacs-dashboard/blob/master/dashboard-widgets.el#L437) section of `dashboard-widgets.el`, but with all the `caddr` calls, I'm finding it kind of hard to decipher. I do note that later in the function, [here](https://github.com/emacs-dashboard/emacs-dashboard/blob/master/dashboard-widgets.el#L443) and [here](https://github.com/emacs-dashboard/emacs-dashboard/blob/master/dashboard-widgets.el#L445), the face for the icon is assigned and I can only surmise that's the section causing the error.

I'm running `GNU Emacs 26.1 (build 1, x86_64-w64-mingw32) of 2018-05-30` on windows.

**Comment by jcs090218** [comment]
I have checked the PR and confirm this as a bug. 

Thank you for reporting this issue to us! I greatly appreciated! 😄

**Comment by jcs090218** [comment]
I have made another PR here #245.

**Comment by jcs090218** [comment]
I have merged the PR #245. Feel free to reopen this if the issue remains! Thank you!
